src/utils/deepLinks.ts

```diff
@@ -1,7 +1,10 @@
 import {Organization} from 'src/client'
+import {OrganizationSummaries} from 'src/client/unityRoutes'
 import {PROJECT_NAME_PLURAL} from 'src/flows'
 
-export const buildDeepLinkingMap = (org: Organization) => ({
+export const buildDeepLinkingMap = (
+  org: Organization | OrganizationSummaries[number]
```

Comment:
👍

You don't have to worry about this, but just for your own information when it comes to software design: I think that I'm going to refactor `buildDeepLinkingMap` to take in an `orgId` rather than an `org`. Right now, the function needs to know the structure of the `Organization` type to function (it needs to know that it has an `id` and how to access it). It shouldn't need to worry about that, since every line in this function only operates on the org id. It would be a cleaner function from a design perspective to pass in `orgId`, since that's all the function needs. This is better because if the structure of `Organization` needs to change, this function won't have to change with it, since it won't care about where org id comes from.

Reply:
That's definitely the right move. It will make the function reusable with other types of structures if we were to limit the argument to `orgId` instead of the org object in its entirety.